tests/usecase/test_db_config_options.py

```diff
@@ -0,0 +1,85 @@
+import os
+from database_config import get_database_config
+
+
+class TestDatabaseConfigOptions:
+    def clean_env(self):
+        """
+        Setting different environment variables for test in the same process
+        can lead to flaky tests.
+        """
+        if "DATABASE_URL" in os.environ.keys():
+            del os.environ["DATABASE_URL"]
+
+        for key in filter(
+            lambda x: x.startswith("MXRUNTIME_Database"),
+            list(os.environ.keys()),
+        ):
+            del os.environ[key]
+
+    def test_no_setup(self):
+        self.clean_env()
+        try:
+            config = get_database_config()
+            assert config is None
+        except Exception as e:
+            assert "Can't find database configuration" in str(e)
+
+    def test_mx_runtime_db_config(self):
+        """
+        Test is MXRUNTIME variables are set up no database configuration is returned
+        based on DATABASE_URL or VCAP_SERVICES
+        """
+        self.clean_env()
+        os.environ["MXRUNTIME_DatabaseType"] = "PostgreSQL"
+        os.environ[
+            "MXRUNTIME_DatabaseJdbcUrl"
+        ] = "postgres://username:password@rdsbroker-testfree-nonprod-1-eu-west-1.asdbjasdg.eu-west-1.rds.amazonaws.com:5432/testdatabase"  # noqa E501
+
+        config = get_database_config()
+        assert config is None
+
+    def test_database_url(self):
+        self.clean_env()
+        os.environ[
+            "DATABASE_URL"
+        ] = "jdbc:postgres://user:secret@host/database"
+
+        config = get_database_config()
+        assert config is not None
```

Comment:
Could you extend this beyond making sure that the config is not `None`? If someone was to remove almost all logic from `get_database_config` and only return an empty string for example, this test would still happily pass, even though the `config` is not what we expect.

Also for the next testcase.

Reply:
yes, but that is not the intension of this test. db specific logic is tested in other test cases.

Reply:
But don't we want to make sure that the configuration is assembled correctly based on environment variables? Just to be sure, could you link those tests?

Reply:
specific cases are test_rds, test_mysql, test_sap_hana. Update the test to check the database type in the returned config.